functions/src/Twilio/text.ts

```diff
@@ -0,0 +1,35 @@
+import * as functions from "firebase-functions";
```

Comment:
Remove unused imports.

Reply:
It is not unused, I need that for the environment config